src/ert/_c_wrappers/enkf/analysis_config.py

```diff
@@ -49,11 +49,7 @@ def __init__(
 
     def _copy_modules(self):
         for element in self._analysis_copy:
-            if isinstance(element, list):
-                src_name, dst_name = element
-            else:
-                src_name = element[ConfigKeys.SRC_NAME]
-                dst_name = element[ConfigKeys.DST_NAME]
+            src_name, dst_name = element
```

Comment:
I take it that we should always get lists of tuples from now on.
But should we then express this in the __init__ using optionals ?

``` python
def __init__(
        self,
        ...
        analysis_copy=None,
        analysis_set_var=None,
        analysis_select=None,
    ):
```

Reply:
what() ?

Reply:
Discussed and was decided not to follow up on.